pillar.example

```diff
@@ -19,3 +19,32 @@ libvirt:
     auth_unix_ro: 'none'
     auth_unix_rw: 'none'
     auth_tcp: 'none'
+
+  tofs:
+    # The files_switch key serves as a selector for alternative
+    # directories under the formula files directory. See TOFS pattern
+    # doc for more info.
+    # Note: Any value not evaluated by `config.get` will be used literally.
+    # This can be used to set custom paths, as many levels deep as required.
+    files_switch:
+      - any/path/can/be/used/here
+      - id
+      - roles
+      - osfinger
+      - os
+      - os_family
+
+    # All aspects of path/file resolution are customisable using the options below.
+    # This is unnecessary in most cases; there are sensible defaults.
+    # path_prefix: template_alt
```

Comment:
@baby-gnu I recently added a couple of lines here in the `template-formula`, based upon a PR we received in the `nginx-formula`.

Reply:
Sorry, it's just above this line (on my mobile device, hit the wrong line!).

Reply:
I just repushed a fixed version.

Thanks.